datadog_checks_dev/tests/tooling/test_release.py

```diff
@@ -42,3 +50,32 @@ def test_get_agent_requirement_line():
         load.return_value = {"supported_os": ["linux", "mac_os"]}
         res = get_agent_requirement_line('foo', '1.2.3')
         assert res == "datadog-foo==1.2.3; sys_platform != 'win32'"
+
+
+def test_update_agent_requirements_with_existing_integration():
```

Comment:
Would it make sense to combine these two tests with `pytest.mark.parametrize`?

Reply:
Less code duplication is a good thing yes. I also slightly modified the dataset